### ISSUE TITLE
Apply for datasets

### ISSUE THREAD

**Issue by sim1n999** [shizhouxing/LLM-Detector-Robustness#1]
Thank you very much for your excellent work. I have a strong interest in your research and I find your methods very inspiring and appreciate that you have open sourced your code. 

However, I noticed that the datasets doesn't seem to be uploaded, can I request access to the dataset used in your research or can I find it elsewhere? I assure you that all data is used strictly for academic purposes and meets any conditions you specify. 

If possible, I would appreciate it if you could upload the data somewhere, or send it to me via email at `siming8741@gmail.com` .

Thank you again for your contribution!

**Comment by shizhouxing** [comment]
Hi @1821518274 ,

Thanks for your interest in our work! Could you elaborate what dataset are you referring to? If you are referring to datasets like xsum, eli5, etc., they are automatically loaded from huggingface `datasets` (https://huggingface.co/docs/datasets/en/index): https://github.com/shizhouxing/LLM-Detector-Robustness/blob/master/dataset/load_data.py

**Comment by sim1n999** [comment]
Thanks for your reply, I was referring to the xsum and eli5 datasets. It seems then that it didn't load correctly because of my bad  network, I will try to download it locally directly from huggingface. Thanks again for your reply and good luck with your work!
